src/service.js

```diff
@@ -0,0 +1,15 @@
+export class Service{
+    constructor(domain) {
+        this.domain = domain;
+    }
+
+    async getCategories(){
+        const req = `${this.domain}categories`;
```

Comment:
@IgorChaikin Since we need to debug code on localhost and work with it on heroku, we can send request to `/categories` without specifying domain name, browser will use current domain name automatically